### PR TITLE
Added a TNT end

### PR DESCRIPTION
**Pull request by BenjaminAmos**
The TNT end contains almost an entire area full of TNT. To test it you can set its spawn chance to 200 and it will certainly spawn. The TNT will explode when used however this can connect rooms sometimes as a result (all the surrounding blocks are destroyed). I have also corrected some spellings.

terasology-170113170513-1920x1200
terasology-170113170538-1920x1200

### CODE REVIEW

File: assets/prefabs/structures/dungeon3x3StoneCorridor/dungeonCooridorTNTEnd.prefab [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "StructureTemplate" : {
+        "type": "GooeysQuests:dungeon3x3StoneCorridor",
```
Comment (flo) [inline assets/prefabs/structures/dungeon3x3StoneCorridor/dungeonCooridorTNTEnd.prefab]: Good job figuring that out

File: assets/prefabs/structures/dungeon3x3StoneCorridor/dungeonCooridorTNTEnd.prefab [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        "front": "FRONT",
+        "spawnChance": 2
+    },
+    "SpawnBlockRegions": {
+        "regionsToFill":[
+            /* First fill everything with air, to make later content placing simpler: */
```
Comment (flo) [inline assets/prefabs/structures/dungeon3x3StoneCorridor/dungeonCooridorTNTEnd.prefab]: this line seems to be a copy and paste mistake

**Comment by BenjaminAmos** [comment]
The changes needed should have been made. When spawned, the room is always connected and the prefab now has a working ground condition added.

**Comment by flo** [comment]
I can't try it out right now but I will tomorrow evening when I find time. Is this for google GCI? If yes what task instance?

**Comment by flo** [comment]
The spawning point of the building is at the very right corner, which makes the corridor ending have an offset which I think is not intended.

Also I noticed that it fills the tnt before a region that does not get filled.

Are you okay with me turning it into a single TNT room that only spawns when there is a chance to blow the way free to a next cave, the outside or another room? Then the player would actually have a reason to blow it up and not just mine it.

**Comment by BenjaminAmos** [comment]
Yes, that should be alright. It might actually give the room a better purpose.

**Comment by flo** [comment]
Merged, thanks for your contribution.

**Pull request merged**